packages/core/src/storages/request_queue_v2.ts

```diff
@@ -145,10 +134,6 @@ export class RequestQueue extends RequestProvider {
                 nextRequestId,
             });
 
-            setTimeout(() => {
```

Comment:
do we still have this in the rq v1 somewhere?

Reply:
Yes https://github.com/apify/crawlee/blob/51cbe9bc6559a144478d0fb52f05957c98ea3a8c/packages/core/src/storages/request_queue.ts#L172-L176